src/gameplay.js

```diff
@@ -36,31 +39,31 @@ const Gameplay = () => {
     }
 
 
-    const writeLeaderboard = () =>
-    {
+    const writeLeaderboard = () => {
         var playerToAdd = {
-            "name" : settings.playerName,
+            "name": settings.playerName,
             "score": score
         }
         return axios.post('http://localhost:5000/leaderboard', playerToAdd)
-        .then(function (response) {
-          console.log(response);
-          return response;
-        })
-        .catch(function (error) {
-          console.log(error);
-          return false;
-        });
+            .then(function (response) {
+                console.log(response);
+                return response;
+            })
+            .catch(function (error) {
+                console.log(error);
+                return false;
+            });
     }
-    
+
     const endGame = () => {
```

Comment:
Thanks for fixing the gameplay logic!

Reply:
No problem! I had to make sure the gameplay was as fun as possible.